server/controllers/user.js

```diff
@@ -63,8 +63,8 @@ function getGroupsForUser(req, res, next) {
     .catch(err => next(err));
 }
 
-function loadUser(req, res, next, userId) {
-  userService.getUser(userId)
+function loadUser(req, res, next) {
```

Comment:
If we are going to load the user making the request, we should load it on all requests, not just the ones with the `/:userId` param